pylint/lint/parallel.py

```diff
@@ -151,7 +156,7 @@ def check_parallel(linter, jobs, files, arguments=None):
             linter.set_current_module(module, file_path)
             for msg in messages:
                 msg = Message(*msg)
-                linter.reporter.handle_message(msg)
+                linter.reporter.handle_message(msg)  # type: ignore # L134 makes linter always have a reporter attribute
```

Comment:
Mentioning line numbers in comments isn't worth it. The code will eventually get updated without the comment.